### PR TITLE
Feature/randomizers

### PR DESCRIPTION
**Pull request by rvolosatovs**
See TheThingsNetwork/go-utils#10

### CODE REVIEW

File: api/gateway/random.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
+package gateway
```
Comment (johanstokking) [inline api/gateway/random.go]: Copyright

File: api/gateway/random_test.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package gateway
```
Comment (johanstokking) [inline api/gateway/random_test.go]: Copyright

File: api/router/random.go [hunk 3]
```diff
@@ -0,0 +1,107 @@
+package router
```
Comment (johanstokking) [inline api/router/random.go]: Copyright

File: api/protocol/random.go [hunk 4]
```diff
@@ -0,0 +1,35 @@
+package protocol
```
Comment (johanstokking) [inline api/protocol/random.go]: Copyright

File: api/protocol/lorawan/random_test.go [hunk 5]
```diff
@@ -0,0 +1,42 @@
+package lorawan
```
Comment (johanstokking) [inline api/protocol/lorawan/random_test.go]: Copyright

File: api/protocol/lorawan/random.go [hunk 6]
```diff
@@ -0,0 +1,178 @@
+package lorawan
```
Comment (johanstokking) [inline api/protocol/lorawan/random.go]: Copyright

File: api/router/random_test.go [hunk 7]
```diff
@@ -0,0 +1,43 @@
+package router
```
Comment (johanstokking) [inline api/router/random_test.go]: Copyright

File: api/protocol/lorawan/random.go [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+func RandomPayload(mType ...MType) []byte {
+	msg := &Message{}
+	msg.initMACPayload()
+
+	var mTypeVal MType
+
+	if len(mType) > 0 {
+		mTypeVal = mType[0]
+	} else {
+		switch rand.Intn(6) {
```
Comment (htdvisser) [inline api/protocol/lorawan/random.go]: MType is enum 0..5, so you can just cast

**Comment by rvolosatovs** [comment]
@htdvisser @johanstokking

**Pull request merged**